WordPress/Classes/ViewRelated/Blog/Blog Dashboard/Cards/Posts/BlogDashboardCardFrameView.swift

```diff
@@ -64,10 +64,33 @@ class BlogDashboardCardFrameView: UIView {
         return containerStackView
     }()
 
+    /// Button displayed to the left of the ellipsis button.
+    /// Displayed when isNew is set to true. Use interaction is disabled.
+    private(set) lazy var newButton: UIButton = {
+        let button = UIButton()
+        button.setTitle("\(Strings.newLabel) ✨", for: .normal)
+        button.setTitleColor(.text, for: .normal)
+        button.titleLabel?.adjustsFontForContentSizeCategory = true
+        button.titleLabel?.font = WPStyleGuide.fontForTextStyle(.subheadline, fontWeight: .regular)
+        button.backgroundColor = Colors.newButtonBackgroundColor
+        button.layer.cornerRadius = Constants.newButtonCornerRadius
+        button.contentEdgeInsets = Constants.newButtonInsets
+        button.isUserInteractionEnabled = false
+        button.accessibilityTraits = .staticText
```

Comment:
I saw a width ambiguity in the inspector, I suggest adding this to fix the issue

<img width="90%" alt="Screen Shot 2023-04-25 at 3 02 31 PM" src="https://user-images.githubusercontent.com/25306722/234284787-86fe367c-2e9d-46af-8f2e-5d0da1f42f8f.png">

```suggestion
        button.accessibilityTraits = .staticText
        button.setContentHuggingPriority(.defaultHigh, for: .horizontal)
```